host/3.0/buster/amd64/dotnet/dotnet-inproc/dotnet-slim.Dockerfile

```diff
@@ -56,4 +56,8 @@ COPY --from=runtime-image [ "/azure-functions-host", "/azure-functions-host" ]
 
 COPY --from=runtime-image [ "/FuncExtensionBundles", "/FuncExtensionBundles" ]
 
-CMD [ "/azure-functions-host/Microsoft.Azure.WebJobs.Script.WebHost" ]
+COPY start_deprecated.sh /azure-functions-host/
+
+RUN chmod +x /azure-functions-host/start_deprecated.sh
+
+CMD ["/azure-functions-host/start_deprecated.sh"]
```

Comment:
are we overriding this command later stages? what is the reason of using CMD here instead of RUN?

Reply:
CMD and RUN work differently. https://stackoverflow.com/questions/37461868/difference-between-run-and-cmd-in-a-dockerfile

Reply:
my bad .. i was thinking this is only needed during build phase .. ignore this comment